.clang-tidy

```diff
@@ -11,21 +12,8 @@ readability-identifier-naming,\
 -clang-analyzer-nullability.NullablePassedToNonnull,\
 -clang-analyzer-nullability.NullReturnedFromNonnull,\
 -clang-analyzer-nullability.NullableReturnedFromNonnull,\
-performance-move-const-arg,\
```

Comment:
Turned off, this was failing on `Linux Android clang-tidy` 
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8799825936871750113/+/u/test:_lint_android_debug_arm64/stdout

This was quasi-enabled in #36221 (since it wasn't added to `WarningsAsErrors` it was never exposed as warnings in CI).

Reply:
cc @gaaclarke

Reply:
@gaaclarke Would it be helpful to have a GitHub issue to add these back?

Reply:
Done: https://github.com/flutter/flutter/issues/113804